wiki/Ranking_Criteria/en.md

```diff
@@ -31,22 +31,17 @@ Glossary
 
 ### Metadata
 
+- **Official sources**: Places directly from the artist or their label such as official websites, CD Scans, or official uploads to websites like Bandcamp, YouTube, or SoundCloud. Third party websites such as wikis, databases (vndb, vgmdb, etc.), or music services (Spotify, iTunes, Amazon, etc.) do not count as official sources.
```

Comment:
Places -> Releases

Reply:
true. but "Places" definitely doesn't work as-is, because "places" don't come from artists.

maybe "Media that the artist or their label provides, such as [...]"